docs/conversion/sakai_20_2-20_3_oracle_conversion.sql

```diff
@@ -3,6 +3,13 @@ ALTER TABLE rbc_evaluation ADD COLUMN status NUMBER(1,0) NOT NULL;
 UPDATE rbc_evaluation SET status = 2;
 -- END SAK-42371
 
+-- START SAK-45137
+UPDATE rbc_rating SET title = '' WHERE title IS NULL;
+ALTER TABLE rbc_rating MODIFY title VARCHAR2(255) NOT NULL;
+UPDATE rbc_rating SET points = 0 WHERE points IS NULL;
+ALTER TABLE rbc_rating MODIFY points NUMBER NOT NULL;
+-- END SAK-45137
```

Comment:
I'm afraid this isn't going to work for Oracle as is for a few reasons:

* n Oracle an empty string (`''`) is treated as `NULL`, so in the update statement you're setting title to `NULL` where title is already `NULL`, which then obviously makes the following statement fail because you're trying to make a column `NOT NULL` but it still has `NULL` values
* When you specify the `NUMBER` datatype with no parameters it defaults to null precision and scale, which isn't appropriate for this column

To avoid the first issue I would normally use a sentinel value, but I'm not sure if that's appropriate here or not. 

For the second issue, the column was originally created as a `FLOAT` so I would suggest keeping at such unless there's a real need to change it to `NUMBER`. If `NUMBER` is actually required, determine the appropriate precision and scale needed and specify it (ex: `NUMBER(10,0)`).

Reply:
Can you just push a commit on with those changes to the oracle bits? the points column can receive 2dp numbers. the title column could just have 'Title' set instead. Somebody will have to edit it later to repair the actual rating, so it just needs to be something.

Reply:
Done